plugins/gt-workflow/commands/gt-cleanup.md

```
@@ -0,0 +1,389 @@
+Scan all local branches, classify them by state (orphaned, closed PR, stale,
+diverged, behind remote, ahead of remote), and offer category-based cleanup
+actions. Complements `/gt-sync` which handles merged branches.
+
+## Input
+
+Optional arguments:
+
+- `--stale-days N` — Override the default 30-day staleness threshold
+- `--dry-run` — Show the audit report without executing any cleanup actions
+
+#$ARGUMENTS
```

markdownlint-cli2 (0.21.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)